homeassistant/components/kodi/media_player.py

```diff
@@ -370,40 +367,27 @@ def state(self) -> MediaPlayerState:
         if self._no_active_players:
             return MediaPlayerState.IDLE
 
-        if self._properties["speed"] == 0:
-            return MediaPlayerState.PAUSED
+        # On first connect, if Kodi already plays any media,
+        # self_properties might be empty
+        if self._properties:
+            if self._properties["speed"] == 0:
+                return MediaPlayerState.PAUSED
```

Comment:
```suggestion
        if self._properties and self._properties["speed"] == 0:
            return MediaPlayerState.PAUSED
```

Reply:
✅ Will be implemented in next commit
(I tried that during my testing, and got a "keyerror" for speed even then. But I just retried that and it works. The cause must have something else)